web/src/drafts.ts

```diff
@@ -109,32 +144,31 @@ export const draft_model = (function () {
 // since we expect bugged drafts will have either been run through
 // this code or else been deleted after 30 (DRAFT_LIFETIME) days.
 let fixed_buggy_drafts = false;
-export function fix_drafts_with_undefined_topics() {
+export function fix_drafts_with_undefined_topics(): void {
```

Comment:
@timabbott this expects a type of invalid draft, where the topic is undefined. Is it still possible to end up in this state? If not, I'll delete it. If so, how would you recommend handling it with typescript?

Reply:
It's a 2-month period of `main` that was affected, and the symptom was pretty bad. I expect there's at least one server that installed/upgraded to that release and then got stuck there due to the person maintaining it being busy. It would be better to wait, and just do whatever the laziest thing typing-wise.

Reply:
I ended up adding to the draft fixing logic beside undefined topics